card-games-vite/src/components/GamesList/GamesList.jsx

```diff
@@ -0,0 +1,20 @@
+import GameCards from "../GameCards/GameCards";
+import "./GamesList.css";
+
+function GamesList({ gameInfo, handleCardLike }) {
+  return (
+    <div className="games-list">
+      {gameInfo.map((game) => {
+        return (
+          <GameCards
+            key={game.id}
```

Comment:
Good job that you've used `id` for the `key` rather than the `index`.

Here is why we shouldn’t use `index` for that: [click here]( https://programmingwithmosh.com/react/why-do-i-need-keys-in-react-lists/#:~:text=React%20recommends%20that%20you%20do,to%20some%20unstable%20component%20behaviour.)